app/src/main/java/com/sdpteam/connectout/map/MapViewFragment.java

```diff
@@ -56,6 +57,10 @@ private void showNewMarkerList(List<Event> eventList) {
             MarkerOptions m = new MarkerOptions().position(e.getGPSCoordinates().toLatLng()).title(e.getTitle());
             map.addMarker(m);
         }
+        // zoom to first event
+        if (!eventList.isEmpty()) {
+            map.moveCamera(CameraUpdateFactory.newLatLngZoom(eventList.get(0).getGPSCoordinates().toLatLng(), 15));
```

Comment:
I would advise that avoiding the use of magic number 15 in this piece of code, might make it clearer.